web/static/js/components/room.jsx

```diff
@@ -24,12 +26,33 @@ class Room extends Component {
     this.props.retroChannel.on("new_idea_received", (newIdea) => {
       this.setState({ ideas: [...this.state.ideas, newIdea] })
     })
+
+    this.props.retroChannel.on("idea_deleted", idea => {
+      this.setState({ ideas: this._removeIdea(this.state.ideas, idea.id)})
+    })
+  }
+
+  _removeIdea(ideas, id) {
+    let index = ideas.map(idea => idea.id).indexOf(id)
```

Comment:
a simpler approach might be to use a filter,

```javascript
return ideas.filter(idea => idea.id !== id)
```

would remove the `-1` check, which is hard to read, and the slice-a-roni.